### PR TITLE
Background and Gradients

### PR DESCRIPTION
**Pull request by Miguelus373**
## Apple homepage clone

In this project we: 
- Used background images, overlays, and gradients
- Used flexbox
- Used fontawesome and google fonts

### CODE REVIEW

**Comment by bertil291utn** [comment]
**Status:** Changes requested 🟠  :shipit:? Not yet.

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below: 

**Navbar**

- [ ] The first eight elements should be links (to other sections or `#`).

image

**Bottom**

- [ ] It contains four elements which are links (to other articles or `#`) ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))
- [ ]  Also this section should be full horizontal

image

**Implementation details**

- [ ] Elements should be aligned with [float](https://css-tricks.com/all-about-floats/) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/)).
  - You are missing float property in your CSS file

[OPTIONAL]

- [ ] Encourage the use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

### Summary

- You have to elemental requirements please guys be careful next time



<hr>


Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Comment by Miguelus373** [comment]
All requested changes were made

File: style.css [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+  html {
+    font-size: 50%;
+  }
+}
+
+@media screen and (min-width: 1380px) {
+  .header {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0 3.5rem;
```
Comment (kitpao) [inline style.css:257]: - [ ] **[MANDATORY]** Please, remove this margin, because your header and your main have different sizes in the screen:
image

**Comment by Ceci007** [comment]
Thanks for the explanations in the review, we just finished the requested changes.
is nice to have some feedback and the explanations are pretty straight forward.

**Comment by krys2fa** [comment]
# Status: Project Approved! 🥇 
Congratulations on making all the requested changes and getting this project approved. 👍 
You made good use of semantic HTML, implemented CSS best practices and utilized Github Flow. 💪 

You can now merge this branch into the master branch and go on with your next project.

💡  Do not forget to submit your project completion form after merging this branch.

All the best team,
Chris

**Pull request merged**